services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/KafkaMessageCollector.java

```diff
@@ -50,37 +47,17 @@
 public class KafkaMessageCollector implements IFloodlightModule {
```

Comment:
Shouldn't we transform it to IService instead?

Reply:
There is no simple answer here. From my point of view FL service (IFloodlightService) is something that can be used "outside" of it i.e. it provide some "function". FL module (IFloodlightModule) is a container for services and "consumer" of "funtions" provided by FL services.

So until there is no any service/module that need to call something implemented inside `KafkaMessageCollector` it can stay IFloodlightModule.

But from other point of view we can convert it into IService(IFloodlightService) just to keep everithing uniformly.